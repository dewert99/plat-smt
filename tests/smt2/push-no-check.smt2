(declare-fun f (Bool) Bool)
(assert (f true))
(push)
(assert (= (f false) (f true)))
(pop)
(assert (not (f false)))
(check-sat)
(get-model)