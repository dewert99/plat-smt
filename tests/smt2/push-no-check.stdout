sat
(
 (define-fun f ((x!0 Bool)) Bool
  (ite (= x!0 false) false
  (ite (= x!0 true) true
   false)))
)
